bin/arnold

```diff
@@ -194,6 +200,7 @@ COMMANDS:
   create_db_vault   create or update a vault file with databases credentials
   htpasswd          initialize a namespace's htpassword files
   info              display current configuration
+  install           fetch dependencies (deployable community apps)
```

Comment:
`install` is not really intuitive. I expect something else then fetching dependencies.

Could be `dependencies` ? `dependency` ?